content/git/concepts/log/log.md

```diff
@@ -0,0 +1,29 @@
+---
+Title: 'Log'
+Description: 'The git log command is used to track the commit history of a project repository.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Developer Tools'
+Tags:
+  - 'Git'
+  - 'Version Control'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'learn-git'
+---
+
+In Git, the **`log`** command is used to track a [branch's](https://www.codecademy.com/resources/docs/git/branch) commit history. After running, it will display project-related information, such as a detailed commit history, which can be used as a reference to go back in time to a previous version of the project.
+
+## Example
+
+Below is a short example of how the `git log` command works. Running `git log` will show a list of all previous commits on the current branch:
+
+```shell
+$ git log
+
+On current branch
+
+commit   f5b5bd8f9eaa443d4020cbe918x742e7ddd22000
+Author : John Doe <John.Doe@mail.com>
+Date: Mon May 22 14:21:03 2023 -0400.
```

Comment:
@RaffyRod Thanks for making that change! Since you decided to change the content, it would be helpful to include a sentence or two about the commit hash and what it means or what can be done with it. Like git revert. Revert isn't on Docs but I can make a note to have this added